blocks/README.md

```diff
@@ -0,0 +1,33 @@
+# Blocks Directory
```

Comment:
It's possible we won't need this directory at all, but it's fine to leave it here for now and we can make that call later